Add more end-user friendly explanation of e-filing (#255)
SuffolkLITLab/docassemble-AssemblyLine-documentation: docs/efiling/overview.md

## Overview

The Suffolk LIT Lab has created a tool to let guided interview authors add
one-click delivery of documents directly to the court's electronic docket.

> This section talks about how to use our tools whether you are:

Reviewer: ```suggestion
You can use use our tool whether you are:
```

answers to a lot of questions the litigant would otherwise need to provide in
traditional e-filing.

In addition to helping with the filing process, our e-filing system allows
litigants to lookup court information, electronically serve the other party, and
to look up information from existing cases.

## Using the LIT Lab's product

The LIT Lab has actually created three products that operate at different layers
to connect with Tyler Technologies and JeffNet:

1. A [REST API](https://github.com/SuffolkLITLab/EfileProxyServer) provided by a
   Java server that speaks directly to Tyler and JeffNet
1. A [Python
   package](https://github.com/SuffolkLITLab/docassemble-EFSPIntegration) that
   provides an object-oriented interface to the REST API
1. A [Docassemble YAML
   file](https://github.com/SuffolkLITLab/docassemble-EFSPIntegration/blob/main/docassemble/EFSPIntegration/data/questions/efiling_integration.yml)
   that uses the Python code to provide an end-to-end filing solution.

Many states are also interested in the accessibility of guided interview that
use an e-filing integration. We have produced a [test
suite](../automated_testing.md) that can provide automated

> accessibility information to audit Docassemble interview compliance with WCAG AA
> and AAA certification levels.

Reviewer: ```suggestion
accessibility information to audit Docassemble interview compliance with WCAG AA 2.0 and 2.1.
```
We can't do AAA with automation I think.
